### DISCUSSION TITLE
milvus能不能提供国产化认证文件

### DISCUSSION THREAD

**Discussion by OpenSourceApp-coder** [milvus-io/milvus#35795] [Q&A and General discussion]
milvus能不能提供国产化认证文件

**Comment by OpenSourceApp-coder** [comment]
milvus能不能提供国产化认证文件？我是中国人，非必须不使用英语，如果看不惯可以不回答。

**Reply by yhmo** [reply]
这个事情我确实不太了解，不好直接回复你。我先找懂得相关领域的人咨询后再回复你。或者你进milvus的微信用户群问也可以。进去milvus微信群的方式是，微信里关注zilliz公众号，然后随便打开一篇文章，拉到底部有个二维码。

**Comment by yhmo** [comment]
milvus是基于Apache 2.0 License的开源项目，按道理不需要提供任何认证文件。其他公司可以把milvus拿去商用，相关认证应该由这些做商业化的公司去做。

举个例子，zilliz基于milvus构建的zilliz cloud就做了安全认证，相关证书在这里可以看到：
https://zilliz.com/security
https://zilliz.com.cn/security
